compiler/dyno/include/chpl/queries/query-impl.h

```diff
@@ -124,14 +124,25 @@ template<typename... ArgTs>
 void Context::queryBeginTrace(const char* traceQueryName,
                               const std::tuple<ArgTs...>& tupleOfArg) {
 
-  if (breakSet || enableDebugTrace) {
+  if ((breakSet || enableDebugTrace)
+      && std::find(queryTraceIgnoreQueries.begin(),
+                   queryTraceIgnoreQueries.end(),
+                   traceQueryName) == queryTraceIgnoreQueries.end()) {
     auto args = queryArgsToStrings(tupleOfArg);
     size_t queryAndArgsHash = hash_combine(hash(traceQueryName), hash(args));
     if (enableDebugTrace) {
-      printf("QUERY BEGIN     %s (", traceQueryName);
+      queryTraceDepth++;
+      // QUERY BEGIN
+      setQueryDepthColor(queryTraceDepth, std::cout);
+      std::cout << queryTraceDepth
+                << " { "
+                << clearTerminalColor() << traceQueryName << " (";
       queryArgsPrint(tupleOfArg);
-      printf(")\n");
-      printf("QUERY + ARGS HASH:    %zu\n", queryAndArgsHash);
+      std::cout << ") ";
+      setTerminalColor(cyan, std::cout);
+      std::cout <<"QUERY+ARGS HASH: 0x"
```

Comment:
I think we should either have this say just `HASH` or honestly, just nothing at all (a hash value is pretty self explanatory IMO).